go/integration_test/processor/create_test.go

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package processor
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+	"slices"
+	"strings"
+	"testing"
+
+	"github.com/Fantom-foundation/Tosca/go/tosca"
+	"github.com/Fantom-foundation/Tosca/go/tosca/vm"
+	"github.com/ethereum/go-ethereum/common"
```

Comment:
do we want to add more dependencies from geth or would it be too much work to implement our own address generator ?

Reply:
I think inside the tests it is fine to have dependencies to geth, we need to ensure that the result is correct somehow. We also have this in other integration test files.

Reply:
I noticed them, and some of them seem easy to remove. but perhaps it should be another task